src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -85,20 +88,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     final Entity commentEntity;
     long lectureId = Long.parseLong(request.getParameter(PARAM_LECTURE));
-    Key lecture = KeyFactory.createKey(LectureUtil.KIND, lectureId);
-    // TODO: Require non-zero content length. See: #183.
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
+    // TODO: Validate that the transcript line contains the timestampMs.
+    long transcriptId = Long.parseLong(request.getParameter(PARAM_TRANSCRIPT_LINE));
```

Comment:
I believe 'transcriptId' should be renamed 'transcriptLineId'. This is the id of the transcript line and not the transcript, right?

Reply:
ah yes, fixed!